extensions/wikia/UserLogin/templates/UserSignupSpecial_WikiaMobileIndex.php

```diff
@@ -1,92 +1,110 @@
+				'label' => wfMessage( 'yourname' )->text(),
+			],
+			[ //actual username field
 				'type' => 'text',
 				'name' => 'userloginext01',
-				'value' => htmlspecialchars($username),
-				'placeholder' => wfMessage('yourname')->text(),
+				'value' => htmlspecialchars( $username ),
+				'placeholder' => wfMessage( 'yourname' )->text(),
 				'isRequired' => true,
-				'isInvalid' => (!empty($errParam) && $errParam === 'username'),
-				'errorMsg' => (!empty($msg) ? $msg : '')
-			),
-			array(
+				'isInvalid' => ( !empty( $errParam ) && $errParam === 'username' ),
+				'errorMsg' => ( !empty( $msg ) ? $msg : '' )
+			],
+			[
 				'type' => 'email',
 				'name' => 'email',
 				'value' => Sanitizer::encodeAttribute( $email ),
-				'placeholder' => wfMessage('email')->text(),
+				'placeholder' => wfMessage( 'email' )->text(),
 				'isRequired' => true,
-				'isInvalid' => (!empty($errParam) && $errParam === 'email'),
-				'errorMsg' => (!empty($msg) ? $msg : '')
-			),
-			array( //fake password field (not in use)
+				'isInvalid' => ( !empty( $errParam ) && $errParam === 'email' ),
+				'errorMsg' => ( !empty( $msg ) ? $msg : '' )
+			],
+			[ //fake password field ( not in use )
 				'type' => 'hidden',
 				'name' => 'password',
 				'value' => '',
-				'label' => wfMessage('yourpassword')->text(),
-			),
-			array( //actual password field
+				'label' => wfMessage( 'yourpassword' )->text(),
+			],
+			[ //actual password field
 				'type' => 'password',
 				'name' => 'userloginext02',
 				'value' => '',
-				'placeholder' => wfMessage('yourpassword')->text(),
+				'placeholder' => wfMessage( 'yourpassword' )->text(),
 				'isRequired' => true,
-				'isInvalid' => (!empty($errParam) && $errParam === 'password'),
-				'errorMsg' => (!empty($msg) ? $msg : '')
-			),
-			array(
+				'isInvalid' => ( !empty( $errParam ) && $errParam === 'password' ),
+				'errorMsg' => ( !empty( $msg ) ? $msg : '' )
+			],
+			[
+				'type' => 'hidden',
+				'name' => 'wpRegistrationCountry',
+				'value' => '',
+			],
+			[
 				'type' => 'nirvanaview',
 				'controller' => 'UserSignupSpecial',
 				'view' => 'birthday',
 				'isRequired' => true,
 				'class' => 'birthday-group',
-				'isInvalid' => (!empty($errParam) && $errParam === 'birthyear') || (!empty($errParam) && $errParam === 'birthmonth') || (!empty($errParam) && $errParam === 'birthday'),
-				'errorMsg' => (!empty($msg) ? $msg : ''),
-				'params' => array('birthyear' => $birthyear, 'birthmonth' => $birthmonth, 'birthday' => $birthday, 'isEn' => $isEn),
-			),
-			array(
+				'isInvalid' => ( !empty( $errParam ) && $errParam === 'birthyear' ) ||
+					( !empty( $errParam ) && $errParam === 'birthmonth' ) ||
+					( !empty( $errParam ) && $errParam === 'birthday' ),
+				'errorMsg' => ( !empty( $msg ) ? $msg : '' ),
+				'params' => [
+					'birthyear' => $birthyear,
+					'birthmonth' => $birthmonth,
+					'birthday' => $birthday,
+					'isEn' => $isEn
+				],
+			],
```

Comment:
I think of brackets here like I think of braces for statements. They give visual cues for what the scope is. The above reminds me of code like this.
```php
function foo() {
    if ( someCondition) {
    // do stuff
}
}
```
Having the 'params' closing bracket be indented at the same level as the closing bracket for the array which 'params' is a part of looks awkward to me. It's up to you, but that's my perspective as someone reading the code.